src/app/[locale]/forgotPassword/page.tsx

```diff
@@ -0,0 +1,93 @@
+'use client';
+import React, { useState } from 'react';
+
+import Link from 'next/link';
+import Button from '@atoms/button/button';
+import Message from '@atoms/message/message';
+import { useForm, Controller } from 'react-hook-form';
+import FormInput from '@molecules/form-input/form-input';
+import { useTranslations } from 'next-intl';
+
+export default function Page() {
+  const [emailSent, setEmailSent] = useState(false);
+  const t = useTranslations('ForgotPassword');
+
+  const {
+    control,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
```

Comment:
ça sera plus claire comme ça
```suggestion
  } = useForm(
  mode: 'onBlur',
  );
```